guides/devboard_as_flasher.rst

```diff
@@ -0,0 +1,64 @@
+Using an ESP devboard as serial flasher
+=======================================
+
+.. _devboard-as-flasher:
+
+If you have a functioning ESP devboard onto which you can flash firmware via USB port,
+it can be also used to flash/communicate via serial to other devices as well.
+We will refer to the devboard with functional USB_UART bridge chip as flasher board for this guide.
+
+Make sure you've read the :doc:`/guides/physical_device_connection` for properly understanding of the functionality of your flasher devboard.
+
+.. figure:: /guides/images/devboard-as-flasher.png
+    :align: center
+    :width: 75.0%
+
+    Connection diagram for an ESP flash target
+
+You need to make the following electrical connections:
+
```

Comment:
```suggestion
You need to make the following electrical connections:

Note that the 5V connection on either board may be labelled either ``5V`` or ``VIN``. Some boards may not have a 5V connection and will require 3.3V only.

Rather than powering the target board from the flasher board, it is also possible to use a separate power supply, just make sure all the ground pins are connected together.
```